src/test/java/spoon/test/main/MainTest.java

```diff
@@ -38,63 +38,28 @@ public void testGenericContract() throws IOException {
 		Launcher launcher;
 		CtPackage rootPackage;
 
-		// we have to remove the test-classes folder
-		// so that the precondition of --source-classpath is not violated
-		// (target/test-classes contains src/test/resources which itself contains Java files)
-		StringBuilder classpath = new StringBuilder();
-		for (String classpathEntry : System.getProperty("java.class.path").split(File.pathSeparator)) {
-			if (!classpathEntry.contains("test-classes")) {
-				classpath.append(classpathEntry);
-				classpath.append(File.pathSeparator);
-			}
-		}
-		String systemClassPath = classpath.substring(0, classpath.length() - 1);
-
 		launcher = new Launcher();
-
-		launcher.setArgs(new String[] {
-				"-o", "target/spooned",
-				"--destination","target/spooned-build",
-				"--source-classpath", systemClassPath,
-				"--compile", // compiling Spoon code itself on the fly
```

Comment:
Did we lose this flag?

Reply:
They really weren't necessary, we only want to build the model and verify the classes.